src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.8.2" PrivateAssets="All" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" PrivateAssets="All" />

```

Comment:
Don't believe this supports netstandard1.3

Reply:
Yep, that's why I upgraded to 2.0 in the other PR
(I forgot to rebase this change though)